src/store/flow/views/import.ts

```diff
@@ -249,18 +249,6 @@ export const actions: ActionTree<IImportState, IRootState> = {
         isValid = false
       }
 
-      if (flowContainer.flows?.length > 1) {
-        commit('validation/pushAjvErrorToValidationStatuses', {
-          key,
-          ajvError: {
-            dataPath,
-            keyword,
-            message: Lang.trans('flow-builder-validation.importer-currently-supports-single-flow-only'),
-          } as ErrorObject,
-        }, {root: true})
-        isValid = false
-      }
-
```

Comment:
Thanks for this, but after our pairing, we noticed we should add another changes to avoid the flow[0] in https://github.com/FLOIP/flow-builder/blob/master/src/store/flow/utils/importHelpers.ts#L33